common-lib/src/main/scala/com/gu/mediaservice/lib/play/EntityStreamExceptionFilter.scala

```diff
@@ -0,0 +1,19 @@
+package com.gu.mediaservice.lib.play
+
+import akka.http.scaladsl.model.EntityStreamException
+import akka.stream.Materializer
+import com.typesafe.scalalogging.StrictLogging
+import play.api.mvc.{Filter, RequestHeader, Result, Results}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class EntityStreamExceptionFilter(override val mat: Materializer)(implicit ec: ExecutionContext)
```

Comment:
This is a really odd looking filter so perhaps we could add a couple of lines of docs?
A more descriptive name (for purpose rather than function) might be better but can't think of one.

Reply:
`ConnectionBrokenFilter`?